package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "npm-install-peers",
-  "version": "1.2.1",
+  "version": "1.2.2-1",
```

Comment:
Please keep the version bump outside of the PR, I'll update this when I tag the next version.

Reply:
thank you for the review. I have removed it.